### ISSUE TITLE
Bug with email reports for internal errors

### ISSUE THREAD

**Issue by vegu** [peeringdb/peeringdb#1023]
**Describe the bug**

Email report for internal errors appears to get get sent to incorrectly formatted recipient (u@peeringdb.com, u@peeringdb.com`)

This might be as simple as a configuration fix or might require code changes.

Looks like we are using `django.utils.log.AdminEmailHandler` which i believe uses the `ADMINS` setting to determine recipients.

**Comment by vegu** [comment]
As part of #715 we rolled out some fixes to how the `ADMINS` setting is specified (it was definitely broken before), will run some tests to see if that fixed the issue on beta.

During investigation of this we also found that it might make sense to expose a separate setting for an ops specific email to be used to dispatch error messages to. Currently it uses the `SERVER_EMAIL`  setting which is set to support@peeringdb.com.

`SERVER_EMAIL` is also used to specify the sender for emails sent to users, so should not be changed.
